sema/ro/creator/robuilder.py

```
@@ -48,8 +48,17 @@ def _build(self):
         )
 
         for identifier, properties in self._blueprint.body.items():
-            a = properties.get("$type")
+            if identifier == "./":
+                a = "schema:Dataset"
+            elif identifier == "ro-crate-metadata.json":
+                a = "schema:CreativeWork"
+            elif not "://" in identifier:
```

Ruff (0.16.2)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)